### PR TITLE
analytics documentation updated

### PR DESCRIPTION
**Pull request by mdacoca**
### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/administration/advanced/analytics.md [hunk 1]
```diff
@@ -1,24 +1,389 @@
...
+| obj_val         | value for the event as string or None (e.g., frame number, number of added annotations) |
+| source          | who generates the log event (e.g., server, ui)                                          |
+| timestamp       | local event time (in general for UI and server, the time is different)                  |
+| count           | how many times in the row it occurs                                                     |
+| duration        | how much time does it take (it can be 0 for events without duration)                    |
+| project         | project ID or None                                                                      |
+| task            | task ID or None                                                                         |
+| job             | job ID or None                                                                          |
+| user            | user ID or None                                                                         |
+| organization    | organization ID or None                                                                 |
```
Comment (azhavoro) [inline site/content/en/docs/administration/advanced/analytics.md:109]: Please update the table to match the actual database schema https://github.com/opencv/cvat/blob/develop/components/analytics/clickhouse/init.sh

File: site/content/en/docs/administration/advanced/analytics.md [hunk 2]
```diff
@@ -1,24 +1,389 @@
...
+[Analytics Components](https://github.com/opencv/cvat/tree/develop/components/analytics)
+
+### Events Log structure
+
+[Relational database](https://github.com/opencv/cvat/blob/develop/components/analytics/clickhouse/init.sh)
+schema with the following fields:
+
+<!--lint disable maximum-line-length-->
+
+| Mandatory Field | Description                                                                             |
```
Comment (azhavoro) [inline site/content/en/docs/administration/advanced/analytics.md:95]:
```suggestion
| Field | Description                                                                             |
```

File: site/content/en/docs/administration/advanced/analytics.md [hunk 3]
```diff
@@ -1,24 +1,389 @@
...
+| duration        | how much time does it take (it can be 0 for events without duration)                    |
+| project         | project ID or None                                                                      |
+| task            | task ID or None                                                                         |
+| job             | job ID or None                                                                          |
+| user            | user ID or None                                                                         |
+| organization    | organization ID or None                                                                 |
+
+<!--lint enable maximum-line-length-->
+
+Extra fields can be added to the JSON blob.
```
Comment (azhavoro) [inline site/content/en/docs/administration/advanced/analytics.md:113]: This is the description for the missing field `payload`. For example: payload is astringified JSON, which can contain any additional data.

File: site/content/en/docs/administration/advanced/analytics.md [hunk 4]
```diff
@@ -1,24 +1,389 @@
...
+the status of the request can be checked by
+adding the query parameter `query_id `to the request:
+
+```bash
+curl -I --user 'user:pass' https://app.cvat.ai/api/events?job_id=123&query_id=150cac1f-09f1-4d73-b6a5-5f47aa5d0031
+```
+
+Upon successful creation, the server will return a `201 Created` status:
+
+```json
```
Comment (azhavoro) [inline site/content/en/docs/administration/advanced/analytics.md:219]: It's not actually a json

File: site/content/en/docs/administration/advanced/analytics.md [hunk 5]
```diff
@@ -1,24 +1,389 @@
...
+<!--lint enable maximum-line-length-->
+
+Every column name can be used like a filter.
+
+If you want to inspect the value, hover over it and click
+on the eye icon.
+
+### Dashboards update
+
+You can adjust the dashboards. To do this, click on the
```
Comment (azhavoro) [inline site/content/en/docs/administration/advanced/analytics.md:358]: It's good idea to add instruction how to adjust dashboards, let's disscuss offline.

File: site/content/en/images/supported_events.png
Comment (azhavoro) [inline site/content/en/images/supported_events.png:1]: please replace all images with jpeg, png is too big...

**Review by azhavoro — CHANGES_REQUESTED** [review]

File: site/content/en/docs/contributing/development-environment.md [hunk 6]
```diff
@@ -238,11 +248,38 @@ You develop CVAT under WSL (Windows subsystem for Linux) following next steps.
...
   ```bash
   sudo systemctl start redis.service
   sudo systemctl start docker.service
   ```
+
+## CVAT Analytics Ports
+
+In case you cannot access analytics, check if the following ports are open:
+
+```json
```
Comment (azhavoro) [inline site/content/en/docs/contributing/development-environment.md:265]:
```suggestion
```yml
```

File: site/content/en/docs/contributing/development-environment.md [hunk 7]
```diff
@@ -238,11 +248,38 @@ You develop CVAT under WSL (Windows subsystem for Linux) following next steps.
...
+cvat_vector:
+    ports:
+      - '8282:80'
+
+  cvat_clickhouse:
+    ports:
+      - '8123:8123'
+```
+
+**Or** remove the following data from
```
Comment (azhavoro) [inline site/content/en/docs/contributing/development-environment.md:275]:
```suggestion
In addition, you can completely disable analytics if you don't need it by deleting the following data from
```

File: site/content/en/images/dashboard_04.png
Comment (azhavoro) [inline site/content/en/images/dashboard_04.png:1]: png -> jpg

File: site/content/en/images/dashboard_03.jpg
[the remark this answers is not in the document]
Comment (mdacoca) [reply site/content/en/images/dashboard_03.jpg:1]: sure

**Review by azhavoro — APPROVED** [review]

**Pull request merged**